kuzzleio/sdk-go
# Wrap Realtime controller for Android SDK

## What does this PR do ?

Wrap Realtime controller methods for Android SDK in order to take and return JSONObject type when needed.

### How should this be manually tested?

run docker run --rm -it -v "$(pwd)":/go/src/github.com/kuzzleio/sdk-go android-x86 /build.sh
Use internal/wrappers/build/android/app/build/outputs/aar/app-release.aar in an android project and see if you have JSONObject type on the Realtime controller's methods
